new-frontiers-14/frontier-station-14
# AI Tweaks

## About the PR
Makes girders work like tables, this wont effect general play with ho they worked.
Make banners pass into, no longer work as a movable wall.

## Why / Balance
AI Was abused to hell.

## How to test
Give yourself god mod
Spawn few carps, spawn some banners and spawn some girders, wrench the girders off the floor and see how the AI wont ignore them anymore.

Spawn a long wall of glass and see how the AI wont be jumping around like crazy as it used to (Still happen but less now)

## Media

## Breaking changes
The changes don't effect anything other then stooping AI abuse.

**Changelog**
Nothing that effect gameplay other then making the AI harder to abuse.
